storybook/stories/_support/utils.js

```diff
@@ -44,7 +45,7 @@ export const generateWallet = (
     reward: new BigNumber(reward).dividedBy(LOVELACES_PER_ADA),
     createdAt: new Date(),
     name,
-    hasPassword: false,
+    hasPassword: hasPassword || true,
```

Comment:
@DeeJayElly this line will always make `hasPassword` to be `true` as even if you pass in `hasPassword` as false `hasPassword || true` will turn it to `true`.

Reply:
Sorry, its a leftover i forgot to revert back this part. Its fixed now.